index.html

```diff
@@ -1325,6 +1325,7 @@ <h4>Notification</h4>
                                 the Thing Descriptions within the directory using 
                                 `create`, `update`, and `delete` event types.
                             </span>
+                            Clients should ignore any event with a type that is unknown to them.
```

Comment:
- Shouldn't this be part of the TD spec?
- Shouldn't it be an RFC2119 assertion?
- How do we test it?